### PR TITLE
Resend whis function branch

### PR DESCRIPTION
**Pull request by jiangxin1998**
## Why do we need this PR?
* Add searchFunction

## How did you address the issue?
* We use usePlacesWidget to autocompelete search

### CODE REVIEW

File: src/pages/MapPage/components/Map/Map.js [hunk 1]
```diff
@@ -94,170 +100,180 @@ function Map(props) {
...
+            position={place}
+            icon={{
+              url: Searchposition,
+              scaledSize: { width: 32, height: 32 }
+            }}
+            animation={1}
+          />
+        ) : (
+          ''
+        )}
```
Comment (KalleChen) [inline src/pages/MapPage/components/Map/Map.js:275]: Just us '&&' instead of '?' here.

File: src/pages/MapPage/components/SearchBar/SearchBar.js [hunk 2]
```diff
@@ -42,35 +44,111 @@ const useStyles = makeStyles((theme) => ({
...
+          enqueueSnackbar('地址超過搜尋範圍', { varient: 'error' })
+        } else {
+          if (placeName.split('路')[1] === undefined) {
+            setPositionName(placeName)
+          } else {
+            setPositionName(placeName.split('路')[1])
+          }
+          setPlacePosition(Place.geometry.location)
+          setMapCenter(Place.geometry.location)
+        }
```
Comment (KalleChen) [inline src/pages/MapPage/components/SearchBar/SearchBar.js:89]: Maybe add some comment here about what it's used to?

**Review by KalleChen — CHANGES_REQUESTED** [review]

File: src/pages/MapPage/components/SearchBar/SearchBar.js (hunk 2 above)
Comment (jiangxin1998) [reply src/pages/MapPage/components/SearchBar/SearchBar.js:89]: Okay already add !

**Review by KalleChen — APPROVED** [review]
LGTM

**Pull request merged**